documentation/user_docs/docs/manual/Manipulating_and_extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -2,166 +2,367 @@
+   second over ``109-111`` and the third ``113-115``.
+
+
+Optional arguments
+------------------
+
+.. code-block:: matlab
+
+   my_cut = cut (data_source, proj, p1_bin, p2_bin, p3_bin, p4_bin, '-nopix', filename)
+
+
+* ``'-nopix'``
+
+  means that the individual pixel information contributing to the resulting data is NOT retained (at present the default
+  is to retain it, resulting in an output that is an sqw object, whereas using ``'-nopix'`` gives a dnd output).
+* ``filename``
+
+  is a string specifying a full filename (including path) for the data to be stored, in addition to being stored in the
+  Matlab workspace.
+
+Further Examples
+----------------
 
 To take a cut from an existing sqw or dnd object, retaining the existing projection axes and binning:
 
-::
+.. code-block:: matlab
+
+   w1 = cut(w,[],[lo1,hi1],[lo2,hi2],...)
+
+.. note::
+
+   The number of binning arguments need only match the dimensionality of the object ``w`` (i.e. the number of plot
+   axes), so can be fewer than 4.
+
+.. note::
+
+   You cannot change the binning in a dnd object, i.e. you can only set the integration ranges and have to use ``[]``
+   for the plot axis. The only option you have is to change the range of the plot axis by specifying ``[lo1,0,hi1]``
+   instead of ``[]`` (the '0' means 'use existing bin size') Note that Note also that
 
-   w1=cut(w,[],[lo1,hi1],[lo2,hi2],...)
 
+section
+=======
+
+``section`` is an ``sqw`` method, which works like a cut but uses the existing bins of an ``sqw`` object.
+
+.. code-block:: matlab
+
+   wout = section(w, p1_bin, p2_bin, p3_bin, p4_bin)
+
+
+Because it only extracts existing bins, this means that it doesn't need to recompute any statistics related to the
+object itself and is therefore faster and more efficient. However, it has the limitation that it cannot alter the
+projection or binning widths from the original.
+
+The parameters of section are as follows:
+
+* ``w``
+
+  ``sqw`` object(s) to be sectioned as an array (of 1 or more elements)
+
+* ``pN_bin``
+
+  Range of bins specified as bin edges to extract from ``w``.
 
-Note that the number of binning arguments need only match the dimensionality of the object ``w`` (i.e. the number of plot axes), so can be fewer than 4. Note also that you cannot change the binning in a dnd object, i.e. you can only set the integration ranges and have to use ``[]`` for the plot axis. The only option you have is to change the range of the plot axis by specifying ``[lo1,0,hi1]`` instead of ``[]`` (the '0' means 'use existing bin size')
+  There are three valid forms for any ``pN_bin``:
 
-To take a cut from an existing sqw object and change the binning along one or more of the plot axes:
+  * ``[]``, ``[0]``
 
-::
+    Use entire original binning axis.
 
-   w1=cut(w, p1_bin, p2_bin,....)
+  * ``[lo, hi]``
 
+    Range containing bin centres to extract from ``w``
```

Comment:
are there restrictions on lo and hi, so that they must reduce the existing range, or will be ignored if they are outside the existing range